modules/puppetca/puppetca_api.rb

```diff
@@ -2,14 +2,18 @@
 
```

Comment:
The line above has to be changed to `require 'puppetca/puppetca_puppet_cert'` as modules/puppetca/puppetca_main.rb is renamed to modules/puppetca/puppetca_puppet_cert.rb so it can load the feature.

Reply:
`puppetca/puppetca_puppet_cert` is loaded via `plugin_configuration::load_classes`, so we don't need it here. This require can therefore be deleted, wondering why tests didn't fail though since that file does not exist anymore.

Reply:
You are right, deleting it works also.